chcemvediet/apps/anonymization/cron.py

```diff
@@ -146,3 +142,92 @@ def attachment_normalization():
         normalize_using_imagemagic(attachment)
     else:
         skip_normalization(attachment)
+
+def recognize_using_mock(attachment_normalization):
+    normalized = os.path.join(settings.PROJECT_PATH,
+                              u'chcemvediet/apps/anonymization/mocks/recognized.odt')
+    with open(normalized, u'rb') as file:
+        AttachmentRecognition.objects.create(
+            attachment=attachment_normalization.attachment,
+            successful=True,
+            file=ContentFile(file.read()),
+            content_type=content_types.ODT_CONTENT_TYPE,
+            debug=u'Created using mocked abbyyocr11.'
+        )
+    cron_logger.info(u'Recogized attachment_normalizon using mocked abbyyocr11: {}'.format(
+        attachment_normalization))
+
+def recognize_using_ocr(attachment_normalization):
```

Comment:
Skusal som sa zamyslat, ci nevieme nejak vsetky tie funkcie, co robia s externymi commanline toolmi nejak zjednotit, lebo tu mame vela copy pastnuteho kodu. Ale hoc su hodne podobne, maju kopec malich odchylok, takze asi sa nedaju zjednotit do jednej pomocnej funkcie. Bez toho aby ta pomocna funkcia bola az prilis komplikovana. Ak by Ti nieco napadlo, navrhi.

Reply:
Vedel by som si predstavit zjednotit funckie `normalize_using_imagemagic` a `normalize_using_libreoffice`, ktore su odseba odlisne iba nazvom package-u, a command line prikazom. Pri `recognize_using_ocr` je v tom problem ze sa pracuje s `AttachmentNormalization` namiesto `Attachment` a aj s inym typom suboru `.odt` namiest `.pdf`. Mozem sa pokusit to zjednotit a uvidim aky z toho vyjde vysledok.

Reply:
Zatial to asi neriesme. Iba zeby Ta napadlo nieco velmi elegantne. Spajat ina `normalize_using_*` nema zmysel, kedze su len 2.

Reply:
:ok: